aws-lc-fips-sys/builder/cmake_builder.rs

```diff
@@ -91,6 +91,8 @@ impl CmakeBuilder {
 
     #[allow(clippy::too_many_lines)]
     fn prepare_cmake_build(&self) -> cmake::Config {
+        env::set_var("GOCACHE", self.out_dir.join("go-build").as_os_str());
```

Comment:
is `"go-build"` saved off in a constant somewhere? can  we always expect it to exist at this point in the build?

Reply:
* This directory won't exist, but it doesn't need to -- the Go compiler just needs sufficient permission to create it.
* Looking at this again, the would more aptly be named `.cache` or `go-cache`, then the Go compiler can create whatever subdirectories (e.g., `go-build`) it needs.